src/components/Calculator.css

```diff
@@ -52,14 +53,21 @@ body {
   color: aliceblue;
 }
 
+h3 {
+  color: darkblue;
+  text-align: center;
+  padding-right: 200px;
+  padding-left: 200px;
+}
+
```

Comment:
- [ ] Kindly make sure to align correctly the paragraph on the homepage as it looks on the provided wireframe designs.

    ## YOUR DESIGN
   ![chrome-capture-2023-8-21](https://github.com/bhushan354/Math-Magician/assets/87197412/87f34069-468d-4dee-bc85-4a754235a4a1)

    ## PROVIDED DESIGN

    ![chrome-capture-2023-8-21 (1)](https://github.com/bhushan354/Math-Magician/assets/87197412/b39ad79b-a085-40ec-8398-ec2fd69b3569)